assets/src/application/ui/layout/espressoTable/ResponsiveTable.tsx

```diff
@@ -9,9 +9,9 @@ import Table from './Table';
 import TableHeader from './TableHeader';
 import TableBody from './TableBody';
 import TableFooter from './TableFooter';
-import './style.scss';
 
-import './tablet-style.scss';
+import './style/index.scss';
+import './style/tablet-style.scss';
```

Comment:
this was imported here but not in `style.scss` because responsive styles need to be imported latest